### DISCUSSION TITLE
I really like this

### DISCUSSION THREAD

**Discussion by biollas** [erew123/alltalk_tts#257] [General]
It has to be the most straight forward, well documented open source project I've ever used. Nothing else I can think of really even comes close.

I just wish it was more straightforward to load trained models. Having to move things around in the xttsv2_2.0.2 directory is cumbersome and gets difficult to keep track of... seems like something that would be easy to just make a dropdown menu for. But what do I know?

**Comment by erew123** [comment]
Hi @biollas 

Please see the BETA version of AllTalk v2 https://github.com/erew123/alltalk_tts/discussions/245

All resolved in that.

Thanks

**Reply by biollas** [reply]
Very cool, thanks!

**Comment by erew123** [comment]
Also screenshots here https://github.com/erew123/alltalk_tts/discussions/237

**Comment by gshawn3** [comment]
+1 on the excellent documentation, you really did a fantastic job on it!

**Reply by erew123** [reply]
Glad someone has read it :) hah. Thanks though!
